packages/gestalt/src/TextField/InternalTextField.css

```diff
@@ -36,7 +52,34 @@
 .textFieldWrapper {
   display: flex;
   flex-wrap: wrap;
-  padding-bottom: var(--space-100);
+}
+
+/*
+Overriding the default bottom padding with important because tags need less padding at the bottom
+*/
+
+.sm.textFieldWrapper {
+  /* The tag's margin bottom adds the other 2px */
+  /* stylelint-disable-next-line declaration-no-important */
+  padding-bottom: var(--space-0) !important;
```

Comment:
Are these `!important`s needed? Can this selector be improved to take precedence without it?

Reply:
Not sure at the moment, because default padding of the `TextField` is defined in `.textField.md`. Could be something we revisit

These classes specfically modify the body of the regular TextField w/ tags